_studio/mfx_lib/encode_hw/CMakeLists.txt

```diff
@@ -28,6 +28,17 @@ foreach( dir ${Encoders} )
   include_directories( ${MSDK_LIB_ROOT}/encode_hw/${dir}/include )
 endforeach()
 
+list( APPEND hevcdir
+  hevc/agnostic
+  hevc/agnostic/g11
+  hevc/linux
+  hevc/linux/g11
+  )
+foreach( dir ${hevcdir} )
+  include_directories( ${MSDK_LIB_ROOT}/encode_hw/${dir} )
```

Comment:
I would wish to see explicit list of include directories here - it's not too big actually

Reply:
done